plugin/lsp.vim

```diff
@@ -110,42 +112,42 @@ command! LspDocumentFold call lsp#ui#vim#folding#fold(0)
+nnoremap <silent> <plug>(lsp-previous-error) :<c-u>execute("call lsp#ui#vim#diagnostics#previous_error()")<cr>
+nnoremap <silent> <plug>(lsp-previous-error-nowrap) :<c-u>execute("call lsp#ui#vim#diagnostics#previous_error("--nowrap")")<cr>
+nnoremap <silent> <plug>(lsp-next-warning) :<c-u>execute("call lsp#ui#vim#diagnostics#next_warning()")<cr>
+nnoremap <silent> <plug>(lsp-next-warning-nowrap) :<c-u>execute("call lsp#ui#vim#diagnostics#next_warning("--nowrap")")<cr>
+nnoremap <silent> <plug>(lsp-previous-warning) :<c-u>execute("call lsp#ui#vim#diagnostics#previous_warning()")<cr>
+nnoremap <silent> <plug>(lsp-previous-warning-nowrap) :<c-u>execute("call lsp#ui#vim#diagnostics#previous_warning("--nowrap")")<cr>
+nnoremap <silent> <plug>(lsp-next-diagnostic) :<c-u>execute("call lsp#ui#vim#diagnostics#next_diagnostic()")<cr>
+nnoremap <silent> <plug>(lsp-next-diagnostic-nowrap) :<c-u>execute("call lsp#ui#vim#diagnostics#next_diagnostic("--nowrap")")<cr>
+nnoremap <silent> <plug>(lsp-previous-diagnostic) :<c-u>execute("call lsp#ui#vim#diagnostics#previous_diagnostic()")<cr>
+nnoremap <silent> <plug>(lsp-previous-diagnostic-nowrap) :<c-u>execute("call lsp#ui#vim#diagnostics#previous_diagnostic("--nowrap")")<cr>
+nnoremap <silent> <plug>(lsp-references) :<c-u>execute("call lsp#ui#vim#references()")<cr>
+nnoremap <silent> <plug>(lsp-rename) :<c-u>execute("call lsp#ui#vim#rename()")<cr>
+nnoremap <silent> <plug>(lsp-type-definition) :<c-u>execute("call lsp#ui#vim#type_definition(0)")<cr>
+nnoremap <silent> <plug>(lsp-type-hierarchy) :<c-u>execute("call lsp#ui#vim#type_hierarchy()")<cr>
+nnoremap <silent> <plug>(lsp-peek-type-definition) :<c-u>execute("call lsp#ui#vim#type_definition(1)")<cr>
+nnoremap <silent> <plug>(lsp-workspace-symbol) :<c-u>execute("call lsp#ui#vim#workspace_symbol('')")<cr>
+nnoremap <silent> <plug>(lsp-document-format) :<c-u>execute("call lsp#ui#vim#document_format()")<cr>
+vnoremap <silent> <plug>(lsp-document-format) :<Home>silent <End>execute("call lsp#ui#vim#document_range_format()")<cr>
+nnoremap <silent> <plug>(lsp-document-range-format) :<c-u>set opfunc=lsp#ui#vim#document_range_format_opfunc<cr>g@
+xnoremap <silent> <plug>(lsp-document-range-format) :<Home>silent <End>execute("call lsp#ui#vim#document_range_format()")<cr>
+nnoremap <silent> <plug>(lsp-implementation) :<c-u>execute("call lsp#ui#vim#implementation(0)")<cr>
+nnoremap <silent> <plug>(lsp-peek-implementation) :<c-u>execute("call lsp#ui#vim#implementation(1)")<cr>
+nnoremap <silent> <plug>(lsp-status) :<c-u>echo lsp#get_server_status()<cr>
+nnoremap <silent> <plug>(lsp-next-reference) :<c-u>execute("call lsp#internal#highlight_references#jump(+1)")<cr>
+nnoremap <silent> <plug>(lsp-previous-reference) :<c-u>execute("call lsp#internal#highlight_references#jump(-1)")<cr>
+nnoremap <silent> <plug>(lsp-signature-help) :<c-u>execute("call lsp#ui#vim#signature_help#get_signature_help_under_cursor()")<cr>
```

Comment:
`<silent>` is needed because without it the `screenrow()` and `screencol()` functions in neovim always return `1`, and shall result in incorrect positioning of floating windows.

As I am not sure which of these functions create floating windows, I used `<silent>` for all of them
<br>
We need to have it here because we cannot expect the user to use <silent> in their mappings.

Reply:
As for the `execute()`, I cannot recall why I added them, so I have removed them in the latest force-push.